<a href="MetScape_2.2.jar">Download MetScape_2.2.jar</a>

* __Plugin Name__
  * MetScape
* __Version__
  * 2.2
* __Description__
  * <p>MetScape provides a bioinformatics framework for the visualization and interpretation of metabolomic data using Cytoscape. MetScape allows users to build and analyze networks of genes and compounds, identify enriched pathways from expression profiling data, and visualize changes in metabolite data. Gene expression and/or compound concentration data can be loaded from file(s) (in CSV, TSV, or Excel formats), or the user can simply enter individual compounds/genes/pathways (using KEGG, EHMN, or Entrez Gene IDs) to build metabolic networks without loading a file. MetScape uses an internal relational database stored at NCIBI that integrates data from KEGG and EHMN.</p><p>For more information contact the MetScape development team: metscape-help@umich.edu</p><p>This work developed at the University of Michigan, National Center for Integrative Biomedical Informatics (NCIBI), under National Institutes of Health Grant #U54DA021519..</p>
* __Category__
  * Network generation
* __Release Date__
  * 2011-06-01
* __Minimum Cytoscape Version__
  * 2.8.1
* __Project URL__
  * http://metscape.ncibi.org/metscape2/
* __Authors__
  * Tim Hull, V. Glenn Tarcea, Alla Karnovsky, Terry E. Weymouth, Barbara R. Mirel, Christopher W. Beecher, and Jing Gao, University of Michigan
* __Contacts__
  * Tim Hull, thully@umich.edu
